src/cli/deploy.rs

```diff
@@ -113,19 +113,23 @@ impl Options {
                     )
                 };
 
+                let config = match conf {
+                    Some(path) => toml::from_str(&read_to_string(path)?)?,
+                    None => Config::default(),
+                };
```

Comment:
`match` is redundant here

Reply:
As opposed to using `map`? Being able to leverage `?` here seems preferable.